test/support/adapter_case.ex

```diff
@@ -2,8 +2,14 @@ defmodule Tesla.AdapterCase do
   defmacro __using__(adapter: adapter) do
     quote do
       @adapter unquote(adapter)
-      @http "http://0.0.0.0:#{Application.get_env(:httparrot, :http_port)}"
-      @https "https://0.0.0.0:#{Application.get_env(:httparrot, :https_port)}"
+      # Needed for SSL test with certificate verification
+      if @adapter == Tesla.Adapter.Ibrowse do
+        @http "http://0.0.0.0:#{Application.get_env(:httparrot, :http_port)}"
+        @https "https://0.0.0.0:#{Application.get_env(:httparrot, :https_port)}"
+      else
+        @http "http://localhost:#{Application.get_env(:httparrot, :http_port)}"
+        @https "https://localhost:#{Application.get_env(:httparrot, :https_port)}"
+      end
```

Comment:
It would be better to override this in Ibrowse tests module

Reply:
Done 🙂